2.0/requirements.txt

```diff
@@ -1,4 +1,4 @@
 gunicorn==19.9.0
-django==2.0.10
-psycopg2==2.7.6.1 --no-binary psycopg2
+django==2.0.12
```

Comment:
It looks like 2.0.13 came out a few hours ago.

See: https://www.djangoproject.com/weblog/2019/feb/12/bugfix-release/

Reply:
Great catch! Surprised how many hotfixes were required for this release.